docs/src/running-tests-js.md

```diff
@@ -79,28 +79,34 @@ Since Playwright runs in Node.js, you can debug it with your debugger of choice
 - Debugging a test from the line number where the `test(..` is defined:
 
   ```bash
-  npx playwright test example.spec.ts:42 --debug
+  npx playwright test example.spec.ts:10 --debug
   ```
 
-<img width="1350" alt="Debugging Tests with the Playwright inspector" src="https://user-images.githubusercontent.com/13063165/197800771-50cb2f39-2345-4153-b4ed-de9fe63ba29b.png" />
-
+<img width="1394" alt="Debugging Tests with the Playwright inspector" src="https://user-images.githubusercontent.com/13063165/212744309-4b7e431b-de2a-45ca-b287-6360124adc33.png" />
 
 Check out our [debugging guide](./debug.md) to learn more about the [Playwright Inspector](./debug.md#playwright-inspector) as well as debugging with [Browser Developer tools](./debug.md#browser-developer-tools).
 
-
 ## Test Reports
 
-The [HTML Reporter](././test-reporters.md#html-reporter) shows you a full report of your tests allowing you to filter the report by browsers, passed tests, failed tests, skipped tests and flaky tests. By default, the HTML report is opened automatically if some of the tests failed.
+The HTML Reporter is a built in reporter which shows you a full report of all your tests that have been ran. It shows the name of the test as well as the file name with the line number of the test next to it as well as how long each test took to run and what browsers they were ran on. You can filter the report by passed tests, failed tests, skipped tests or flaky tests. You can also filter by browsers by clicking on the name of the browser next to a test. You can also search for a particular test or browser using the search bar.
+
+To open the report first run your tests:
+
+```bash
+npx playwright test
+```
+
+By default, the HTML report is opened automatically if some of the tests failed. If you need to open the report manually you can use the following command:
 
 ```bash
 npx playwright show-report
 ```
 
-<img width="963" alt="HTML Report > Test Reports view" src="https://user-images.githubusercontent.com/13063165/189140120-71a6bc41-f921-40be-a7a4-61d44da33b20.png" />
+<img width="1392" alt="HTML Report > Test Reports view" src="https://user-images.githubusercontent.com/13063165/212744633-826cce1b-fab1-455a-8ca7-68867c4d4698.png" />
 
-You can click on each test and explore the tests errors as well as each step of the test.
+You can open a detailed view of each test by clicking on the test name. You can then explore the tests errors as well as expand each step of the test to see the code for that step and how long each step took to run.
 
-<img width="955" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/189140263-0e3ac2d9-4e75-40c8-b29c-58323c8be433.png" />
+<img width="1392" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/212744710-c48a327c-fb68-4c85-adca-2bdedbfcf504.png" />
```

Comment:
we should collapse one of them to show how detailed it is.

Reply:
done